src/lib/word-list.js

```diff
@@ -77,6 +78,8 @@ export default class WordList {
     let key = this._makeItemKey(this.languageCode,targetWord)
     if (create && !this.items[key]) {
       let wordItem = new WordItem({targetWord: targetWord, languageCode: this.languageCode})
+      WordlistController.evt.WORDITEM_UPDATED.pub({dataObj: wordItem, params: {segment: 'common'}})
```

Comment:
I would rather keep the responsibility for dispatching the events in the wordlist controller. Is this really needed here? Everywhere that WordlistController calls this code, it also sends that event.